tests/test_rand.py

```diff
@@ -13,25 +13,31 @@
 
 from OpenSSL import rand
 
-from .util import NON_ASCII, TestCase
+from .util import NON_ASCII, tmpfile
```

Comment:
Not sure -- I'd expect pytest to pick up failures globally as well, but if you look [at the first Travis run](https://travis-ci.org/pyca/pyopenssl/jobs/169862020), it doesn't seem to have noticed. 

```
________________ ERROR at setup of TestRand.test_unicode_paths _________________
file /Users/travis/build/pyca/pyopenssl/tests/test_rand.py, line 219
      def test_unicode_paths(self, tmpfile):
E       fixture 'tmpfile' not found
>       available fixtures: cache, capfd, capsys, doctest_namespace, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.
```

I did have `from .util import NON_ASCII` in that version of the code, but it didn't seem to pick them up. With this line in, the tests pass, but then flake8 gets upset.

(I haven't tried running `pytest --fixtures [testpath]` yet, probably should give that another go.)